aea/aea_builder.py

```diff
@@ -87,6 +87,7 @@
 PathLike = Union[os.PathLike, Path, str]
 
 _default_logger = logging.getLogger(__name__)
+DEFAULT_ENV_DOTFILE = ".env"
```

Comment:
We need to make this configurable in aea-config.yaml. Let;'s open an issue

Reply:
https://github.com/fetchai/agents-aea/issues/1811